### PR TITLE
master

### PR DESCRIPTION
**Pull request by shuodata**

### CODE REVIEW

**Comment by femtotrader** [comment]
Hi @shuodata 

Thanks for your contribution.
Please tell me when you consider it's ready for review/merging.

Kind regards

**Comment by shuodata** [comment]
Hi @femtotrader,

It's ready, I just made a small change to allow Trailing stops when opening a position

Regards

File: trading_ig/rest.py [hunk 1]
```diff
@@ -497,13 +498,23 @@ def create_open_position(self, currency_code, direction, epic, expiry,
...
+
+        # Trailing stop is supported in version 2
+        # Version headers should be include
+
+        self.crud_session.HEADERS['LOGGED_IN']['Version'] = '2'
         response = self._req(action, endpoint, params, session)
+        if 'Version' in self.crud_session.HEADERS['LOGGED_IN']:
+            del self.crud_session.HEADERS['LOGGED_IN']['Version']
 
+        # Remove the header to back copatibility
```
Comment (femtotrader) [inline trading_ig/rest.py]: copatibility -> compatibility

**Comment by femtotrader** [comment]
Thanks @shuodata for your contribution but could you fix the small typo I mentioned in https://github.com/ig-python/ig-markets-api-python-library/pull/95#discussion-diff-264980535R517

File: trading_ig/rest.py (hunk 1 above)
Comment (shuodata) [reply trading_ig/rest.py]: fixed!

**Comment by femtotrader** [comment]
Thanks @shuodata

**Pull request merged**